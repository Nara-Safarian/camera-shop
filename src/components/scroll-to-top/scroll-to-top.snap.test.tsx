import { render } from '@testing-library/react';
import ScrollToTop from './scroll-to-top';
import { BrowserRouter } from 'react-router-dom';

window.scrollTo = jest.fn();

test('ScrollToTop snapshot should be rendered correctly', () => {
  const {container} = render(
    <BrowserRouter>
      <ScrollToTop />
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
