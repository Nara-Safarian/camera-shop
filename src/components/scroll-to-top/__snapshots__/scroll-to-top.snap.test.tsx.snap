// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop snapshot should be rendered correctly 1`] = `<div />`;
